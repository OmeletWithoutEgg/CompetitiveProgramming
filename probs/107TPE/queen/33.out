670722.13 -42309775.51
