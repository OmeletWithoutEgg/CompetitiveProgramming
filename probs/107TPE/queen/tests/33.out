-31422525.05 90544951.88
